### ISSUE TITLE
Bug: SK doesn't play nice in multiple windows workflow

### ISSUE THREAD

**Issue by iSeeU816** [brookhong/Surfingkeys#1259]
Hello,

I'm using Surfingkeys (SK) extension all the time and I'm loving it. Thank you!

**TL;DR:** Only read `Steps to reproduce the bug` section.

## Intro

I'm writing this to report a bug in the extension that related on how SK handle
multiple windows when switching between them. Suppose you in window 1 where SK
function very well; then you switch to window 2, calling some of SK commands do
nothing.

I debugged the extension by building an older versions based on Git commits.
After a few tests I figured that v0.9.54 (https://github.com/brookhong/Surfingkeys/commit/664d04226c87dd007acee003534cf90803eae38e) is the cause of this bug or it might be the root cause of the bug going forward.

## Steps to reproduce the bug

I will use local links to make the steps intact and doesn't depend to specific
online links that might be broken later on.

1. **Window 1:**
   1. Open a tab and drag
      `<where_you_keep_surfingkeys_src>/Surfingkeys/dist/documentation/index.html`
      file to open it.
   2. Open `Omnibar` link in the left sidebar by clicking on it or use SK to do it.
   3. Now you have two tabs in window 1.
   4. Go back to `index.html` tab, and duplicate it `yt` then open the duplicate
      tab in its own window `W`.
2. **Window 2:**
   1. Open `StatusBar` link that exist in the left sidebar of the page in a new tab.
3. So far, you have two windows 1 and 2.
   - In window 1 you have `index` and `Omnibar` tabs.
   - And in window 2 you have `index` and `StatusBar` tabs opened.
4. In **window 2**, call `oh` to open `history` in omnibar. Hit `esc` to dismiss it.
5. Go to **window 1**, calling `oh` do nothing!
   - I think all commands that related to omnibar do nothing.
   - A few commands still work, like `R`, `r`, …etc.
   - One of the ways to make the commands `oh`, `T`, `b` and other commands to
     work is to refresh the page.
 6. Still in **window 1**, refresh the page to make SK response to any commands you call.
 7. Go to **window 2**, calling `oh` do nothing!
    - As you can see, only one window response to SK commands at a time.
    - This not only occur between two windows, but any number of windows.

## Closing marks

My workflow of browsing the web is to use many windows; with this bug, I get
very annoyed that the command I call doesn't work, so that pause me a second or
two to realize that I have to refresh the page to make SK response to the
commands I throw at it.

Please fix the annoying bug.

Thank you!

## Software info

- macOS Mojave 10.14.6.
- Brave Browser 80.1.4.96.
  - Also tested it in Google Chrome 83.0.4103.106.
- Surfingkeys 0.9.65 (https://github.com/brookhong/Surfingkeys/commit/9772791e69c15d0b7ae65d588fb68c8a7fc2a794).

**Comment by iSeeU816** [comment]
I have been using this fix for a few hours and it's working OK.

Thank you!
